### ISSUE TITLE
Cannot build rom rev48 with ca65 2.18 : "ld65: Error: Cannot use -Ln twice"

### ISSUE THREAD

**Issue by FredBezies** [X16Community/x16-emulator#298]
Hello.

I tried to build this morning rom rev 48 on my archlinux using ca65 2.18 and I got this error message:

```
INFO: unique module name text_input_ did not appear map file modules. We're going to wing it...
INFO: we're going to go with text_input_CODE_000B.
INFO: unique module name match_ did not appear map file modules. We're going to wing it...
INFO: we're going to go with match_BSS_0001_CODE_01C5.
ld65: Error: Cannot use -Ln twice
make: *** [Makefile:404: build/x16/basic.bin] Error 1
```

Any idea on how to fix this annoying issue? Thanks!

**Comment by mooinglemur** [comment]
Version 2.18 might be too old.  I would recommend building the current git master of cc65 and using that, or at least try with 2.19.

**Comment by FredBezies** [comment]
I tried with cc65 2.19 and it is the same issue. Strangely when I try to build on the last commit, it worked perfectly. Strange.

**Comment by mooinglemur** [comment]
Cannot reproduce with the current top of tree cc65, which I suspect will be the only one we test against and support until a proper version is released beyond 2.19, which was all the way back in 2020.

If you still have an issue with the most current cc65, please open a new issue.
